xostor.md: add separate section for adding disks (#366)
xcp-ng/xcp-ng-org: docs/xostor/xostor.md

```
For a short description, this command (re)creates a PBD, opens DRBD/LINSTOR ports, starts specific services and adds the node to the LINSTOR database.

If you have storage devices to use on the host, a LINSTOR storage layer is not directly added to the corresponding node.

> You can follow the section below to add storage to this new node.

Reviewer: Perhaps you should add a link directly to the next section.

> You can follow [this documentation](#how-to-add-storage-on-a-new-host) to add storage to this new node.

Reviewer 2: How about 'Follow the section below', since we're pointing at the instructions that come right after? That would make the meaning a little more explicit

Reviewer: I disagree with removing the link. :p If we add a new section between the two existing ones, the information given becomes inconsistent.

Reviewer 2: I agree with @Wescoeur. There isn't any drawback in keeping the link, but the text should still point out that the target section is right below

Reviewer 2: @Wescoeur Yes, keeping the link is useful so we should keep it. What about mentioning that the target section comes immediately after? If we add another section later, we could remove the mention then

Reviewer 2: I'm OK with your solution. We can't go wrong with it

### How to add storage on a new host

> On a existing host without storage, you can easily add storage by adding a new Linstor Storage Pool for the Linstor node.

Reviewer: Too much redundancy :sweat_smile: Can we remove this sentence? The two steps are already clear, no?

╰──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```

> 1. Creating VG/LV

Reviewer: ```suggestion
1. Create a LVM volume group and/or thin device
```

> To add disks to the linstor SR, you will need to create VG.

Reviewer: ```suggestion
To add disks to the linstor SR, you will need to create a LVM volume group.
```

Connect to the machine to modify and use `vgcreate` with the wanted disks to create a VG group on the host:
```
vgcreate <GROUP_NAME> <DEVICES>
```

> In our example where we want to use `/dev/nvme0n1` with the group `linstor_group`:

Reviewer: ```suggestion

In our example where we want to use `/dev/nvme0n1` with the group `linstor_group`:
```

lvchange -ay <GROUP_NAME>/<LV_THIN_VOLUME>
```

Most of the time and in our example, we will have:
- `<GROUP_NAME>`: `linstor_group`.
- `<LV_THIN_VOLUME>`: `thin_device`.
- `<SP_NAME>`: `xcp-sr-linstor_group_thin_device`.

> 2. Add a new SP for the node

Reviewer: ```suggestion
2. Create a new storage pool attached to the node
```

> Run the corresponding command on the host where the controller is running to add the Volume Group in the LINSTOR database:

Reviewer: ```suggestion
Run the corresponding command on the host where the controller is running to add the volume group in the LINSTOR database:
```
